lms/djangoapps/courseware/views/index.py

```diff
@@ -387,10 +388,17 @@ def _create_courseware_context(self):
         Returns and creates the rendering context for the courseware.
         Also returns the table of contents for the courseware.
         """
+        original_course = None
+
+        if hasattr(self.course.id, 'ccx'):
+            c = CustomCourseForEdX.objects.get(pk=self.course.id.ccx)
+            original_course = c.course_id
```

Comment:
Either rename original_course to original_course_id or send whole course object.